<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="6" total="6" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="116670" started-at="2014-11-24T20:09:02Z" finished-at="2014-11-24T20:10:59Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="116670" started-at="2014-11-24T20:09:02Z" finished-at="2014-11-24T20:10:59Z">
      <class name="Tests.WhenUserDisplayedAddNewTextMessageView">
        <test-method status="PASS" signature="setUp()[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="setUp" is-config="true" duration-ms="64706" started-at="2014-11-24T21:09:02Z" finished-at="2014-11-24T21:10:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="CheckboxInFilterShouldBeLinked" duration-ms="4336" started-at="2014-11-24T21:10:07Z" data-provider="Filter" finished-at="2014-11-24T21:10:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Entire Alumni Program]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [1] but found [0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [1] but found [0]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:165)
	at Tests.WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(WhenUserDisplayedAddNewTextMessageView.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckboxInFilterShouldBeLinked -->
        <test-method status="PASS" signature="BackToWallPage()[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="BackToWallPage" is-config="true" duration-ms="1" started-at="2014-11-24T21:10:13Z" finished-at="2014-11-24T21:10:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BackToWallPage -->
        <test-method status="PASS" signature="ScreenshotWhenFail(org.testng.ITestResult)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="ScreenshotWhenFail" is-config="true" duration-ms="2959" started-at="2014-11-24T21:10:13Z" finished-at="2014-11-24T21:10:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=CheckboxInFilterShouldBeLinked status=FAILURE method=WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ScreenshotWhenFail -->
        <test-method status="FAIL" signature="CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="CheckboxInFilterShouldBeLinked" duration-ms="4069" started-at="2014-11-24T21:10:16Z" data-provider="Filter" finished-at="2014-11-24T21:10:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Entire Family Program]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [0]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:165)
	at Tests.WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(WhenUserDisplayedAddNewTextMessageView.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckboxInFilterShouldBeLinked -->
        <test-method status="PASS" signature="BackToWallPage()[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="BackToWallPage" is-config="true" duration-ms="1" started-at="2014-11-24T21:10:21Z" finished-at="2014-11-24T21:10:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BackToWallPage -->
        <test-method status="PASS" signature="ScreenshotWhenFail(org.testng.ITestResult)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="ScreenshotWhenFail" is-config="true" duration-ms="2678" started-at="2014-11-24T21:10:21Z" finished-at="2014-11-24T21:10:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=CheckboxInFilterShouldBeLinked status=FAILURE method=WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ScreenshotWhenFail -->
        <test-method status="FAIL" signature="CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="CheckboxInFilterShouldBeLinked" duration-ms="4232" started-at="2014-11-24T21:10:24Z" data-provider="Filter" finished-at="2014-11-24T21:10:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[People you are following]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [1] but found [0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [1] but found [0]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:165)
	at Tests.WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(WhenUserDisplayedAddNewTextMessageView.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckboxInFilterShouldBeLinked -->
        <test-method status="PASS" signature="BackToWallPage()[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="BackToWallPage" is-config="true" duration-ms="1" started-at="2014-11-24T21:10:29Z" finished-at="2014-11-24T21:10:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BackToWallPage -->
        <test-method status="PASS" signature="ScreenshotWhenFail(org.testng.ITestResult)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="ScreenshotWhenFail" is-config="true" duration-ms="3026" started-at="2014-11-24T21:10:29Z" finished-at="2014-11-24T21:10:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=CheckboxInFilterShouldBeLinked status=FAILURE method=WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ScreenshotWhenFail -->
        <test-method status="FAIL" signature="CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="CheckboxInFilterShouldBeLinked" duration-ms="4067" started-at="2014-11-24T21:10:32Z" data-provider="Filter" finished-at="2014-11-24T21:10:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[People following you]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [0]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:165)
	at Tests.WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(WhenUserDisplayedAddNewTextMessageView.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckboxInFilterShouldBeLinked -->
        <test-method status="PASS" signature="BackToWallPage()[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="BackToWallPage" is-config="true" duration-ms="1" started-at="2014-11-24T21:10:37Z" finished-at="2014-11-24T21:10:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BackToWallPage -->
        <test-method status="PASS" signature="ScreenshotWhenFail(org.testng.ITestResult)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="ScreenshotWhenFail" is-config="true" duration-ms="2756" started-at="2014-11-24T21:10:37Z" finished-at="2014-11-24T21:10:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=CheckboxInFilterShouldBeLinked status=FAILURE method=WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ScreenshotWhenFail -->
        <test-method status="FAIL" signature="CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="CheckboxInFilterShouldBeLinked" duration-ms="5659" started-at="2014-11-24T21:10:40Z" data-provider="Filter" finished-at="2014-11-24T21:10:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Only with men]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [3] but found [0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [3] but found [0]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:165)
	at Tests.WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(WhenUserDisplayedAddNewTextMessageView.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckboxInFilterShouldBeLinked -->
        <test-method status="PASS" signature="BackToWallPage()[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="BackToWallPage" is-config="true" duration-ms="1" started-at="2014-11-24T21:10:46Z" finished-at="2014-11-24T21:10:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BackToWallPage -->
        <test-method status="PASS" signature="ScreenshotWhenFail(org.testng.ITestResult)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="ScreenshotWhenFail" is-config="true" duration-ms="4132" started-at="2014-11-24T21:10:46Z" finished-at="2014-11-24T21:10:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=CheckboxInFilterShouldBeLinked status=FAILURE method=WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ScreenshotWhenFail -->
        <test-method status="FAIL" signature="CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="CheckboxInFilterShouldBeLinked" duration-ms="3980" started-at="2014-11-24T21:10:50Z" data-provider="Filter" finished-at="2014-11-24T21:10:54Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Select specific followers]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [0]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:165)
	at Tests.WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(WhenUserDisplayedAddNewTextMessageView.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckboxInFilterShouldBeLinked -->
        <test-method status="PASS" signature="BackToWallPage()[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="BackToWallPage" is-config="true" duration-ms="0" started-at="2014-11-24T21:10:55Z" finished-at="2014-11-24T21:10:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BackToWallPage -->
        <test-method status="PASS" signature="ScreenshotWhenFail(org.testng.ITestResult)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="ScreenshotWhenFail" is-config="true" duration-ms="2713" started-at="2014-11-24T21:10:55Z" finished-at="2014-11-24T21:10:58Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=CheckboxInFilterShouldBeLinked status=FAILURE method=WhenUserDisplayedAddNewTextMessageView.CheckboxInFilterShouldBeLinked(java.lang.String, java.lang.Integer)[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ScreenshotWhenFail -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:Tests.WhenUserDisplayedAddNewTextMessageView@f4721c7]" name="tearDown" is-config="true" duration-ms="1208" started-at="2014-11-24T21:10:58Z" finished-at="2014-11-24T21:10:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Tests.WhenUserDisplayedAddNewTextMessageView -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
